lib/matplotlib/backends/web_backend/mpl.js

```diff
@@ -454,6 +454,18 @@ mpl.findpos = function(e) {
     return {"x": x, "y": y};
 };
 
+/*
+ * return a copy of an object with only non-object keys
+ * we need this to avoid circular references
+ * http://stackoverflow.com/a/24161582/3208463
+ */
+function simpleKeys (original) {
```

Comment:
The stackoverflow link helps, but perhaps an example of the kind of thing this might end up looking like might be a good idea.